suite-native/assets/src/components/AssetItem.tsx

```diff
@@ -74,13 +74,13 @@ export const AssetItem = ({
                         </Box>
                     </Box>
                     <Box alignItems="flex-end">
-                        <DiscreetText>{FiatAmountFormatter.format(fiatBalance)}</DiscreetText>
-                        <DiscreetText color="gray600" typography="hint">
-                            {CryptoAmountFormatter.format(cryptoCurrencyValue, {
-                                isBalance: true,
-                                symbol: cryptoCurrencySymbol,
-                            })}
-                        </DiscreetText>
+                        <FiatAmountFormatter
+                            value={isTestnet(cryptoCurrencySymbol) ? null : fiatBalance}
```

Comment:
I would send value and symbol to formatter and handle logic there.

Reply:
changed in ba28d5486